src/theme/spacing.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable sort-keys */
 export const spacing = {
+  xxxs: '.25rem', // 4px
```

Comment:
Sigh. I do hate the naming, but I don't have a naming convention that I don't hate. We could name it like the xxl versions, but I didn't like that approach either.

E.g. 
```suggestion
  xxs2: '.25rem', // 4px
```

Reply:
Yeah, it did not feel good at all to type that out. Not the least bit proud of it. 

Following the xxl approach is less lame but more confusing in my opinion because the integer could be interpreted as smaller or larger

I mean, we could go with "tiny" or "smallest" but those break the pattern too.